lib/ostruct.rb

```diff
@@ -73,6 +73,7 @@
 # of these properties compared to using a Hash or a Struct.
 #
 class OpenStruct
+  attr_reader :recursive
```

Comment:
This prohibits adding `recursive` member.

Reply:
Thank you so much for pointing! I've updated the PR, Can you please check now?

Reply:
I don't think you can add any method to `OpenStruct`, as it could be incompatible if someone has data with the same name. You should remove the `attr_reader` altogether.